templates/ts-apollo-mongo-apache-kafka-backend/src/util.ts

```diff
@@ -0,0 +1,5 @@
+export function getKafkaTopicName(modelName: string): string {
+  const applicationName = process.env.APPLICATION_NAME || 'graphback-kafka-template-app';
+
+  return `${applicationName}.${modelName}`;
```

Comment:
`_` or `-`  is preferred due to clash with ACL/auth schemes (and web formats)

Reply:
This is actually good delimiter etc. just having context `-` will be better

Reply:
Sure - I was doing some reading on naming conventions and came to this conclusion, but easy change :)

Reply:
Blog post: https://devshawn.com/blog/apache-kafka-topic-naming-conventions/

TL;DR something like `<data-center>.<domain>.<classification>.<description>.<version>`

Reply:
ok.. with that as well. I will see how this will work with auth later. so let;'s merge this this way.